server/app.js

```diff
@@ -34,14 +34,26 @@ app.use(function(err, req, res, next) {
 
 module.exports = app;
 
-var port = normalizePort(process.env.PORT || '3000');
-app.set('port', port);
+// PORT
+var http_port = normalizePort(process.env.PORT || '80');
```

Comment:
I'm not sure if it is a good way to use 80(or 443) port. It's might already be being used in most of the systems.